src/main/java/ch/njol/skript/expressions/ExprSaturation.java

```diff
@@ -40,7 +40,7 @@
 @Name("Saturation")
 @Description("The saturation of a player. If used in a player event, it can be omitted and will default to event-player.")
 @Examples("set saturation of player to 20")
-@Since("2.2-Fixes-v10, 2.2-dev35 (fully modifiable)")
+@Since("2.2-Fixes-v10, 2.2-dev35 (fully modifiable), INSERT VERSION (toggle support)")
 public class ExprSaturation extends PropertyExpression<Player, Number> {
```

Comment:
acceptChange doesn't accept TOGGLE changer

Reply:
It does, it accepts every mode except the ``REMOVE_ALL``